### PR TITLE
Design/password reset

### PR DESCRIPTION
**Pull request by nicolasbarb**

### CODE REVIEW

File: easyselling/Scenario/AuthenticationScenario/AccountCreation/AccountCreationView.swift [hunk 1]
```diff
@@ -12,25 +12,51 @@ struct AccountCreationView: View {
     @ObservedObject var viewModel: AccountCreationViewModel
 
     var body: some View {
+        ZStack {
+            GeometryReader { proxy in
```
Comment (MaxenceMottard) [inline easyselling/Scenario/AuthenticationScenario/AccountCreation/AccountCreationView.swift:16]: Maybe create a component for background image. It's used 2 times

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

**Review by MaxenceMottard — DISMISSED** [review]

File: easyselling/Additions/UI/Components/ImagedBackground.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//
+//  Created by Nicolas Barbosa on 22/12/2021.
+//
+
+import SwiftUI
+
+struct ImagedBackground<Content : View>: View {
+    let content: Content
+
+    init(@ViewBuilder contentBuilder: () -> Content) {
```
Comment (PierreGourgouillon) [inline easyselling/Additions/UI/Components/ImagedBackground.swift:13]: A quoi ca sert ?

File: easysellingTests/ViewModels/Authentication/PasswordResetRequestViewModel_Specs.swift [hunk 3]
```diff
@@ -15,6 +15,17 @@ class PasswordResetRequestViewModel_Specs: XCTestCase {
         await whenRequestingPasswordReset(of: "")
         thenError(is: .emptyEmail)
     }
+
+    // need to find a way to test error nil during asynchronous process
+//    func test_Unshows_error_if_user_has_pressed_reset_again() async {
+//        var verificator: EmailVerificator = FailingEmailVerificator(error: .emptyEmail)
+//        givenViewModel(verificator: verificator)
+//        await whenRequestingPasswordReset(of: "")
```
Comment (PierreGourgouillon) [inline easysellingTests/ViewModels/Authentication/PasswordResetRequestViewModel_Specs.swift:23]: Tu ne delete pas ?

**Review by PierreGourgouillon — APPROVED** [review]

File: easysellingTests/ViewModels/Authentication/PasswordResetRequestViewModel_Specs.swift (hunk 3 above)
Comment (nicolasbarb) [reply easysellingTests/ViewModels/Authentication/PasswordResetRequestViewModel_Specs.swift:23]: Nan il faut que je trouve un moyen de faire passer ce test
Il est bon mais comme on fait du async await je trouve pas de moyen de faire ce que je veux
